test/integration/full/preload-cssom/preload-cssom.js

```diff
@@ -429,23 +426,17 @@ describe('preload cssom integration test', function() {
     commonTestsForRootNodeAndNestedFrame();
   });
 
-  describe('tests for nested document', function() {
-    before(function() {
-      if (!nestedFrame || isIE11) {
```

Comment:
Is there a chance that `nestedFrame` will ever be `false` that we need the if statement still?

Reply:
I have no idea why this was ever there. It wasn't necessary. (otherwise the tests would have failed)